core/src/consensus/stake/actions.rs

```diff
@@ -234,9 +236,16 @@ impl Encodable for Action {
             }
             Action::SelfNominate {
                 deposit,
+                public,
+                signature,
                 metadata,
             } => {
-                s.begin_list(3).append(&ActionTag::SelfNominate).append(deposit).append(metadata);
+                s.begin_list(3)
```

Comment:
You should change the item count also for here.

Reply:
Thanks. I changed it.